src/components/screens/login-screen.jsx

```diff
@@ -46,11 +45,15 @@ function LoginScreen(): Node {
                           onPinChange={(event: any): void => setPin(event.target.value)}
                           onPasswordChange={(event: any): void => setPassword(event.target.value)}
                         />
-
                         <Message error>
-                            {' '}
-                            {t('invalid-login')}
-                            {' '}
+                            <List>
+                                <List.Item>
+                                    <Image src={Subtrack} className="Subtrack" alt="Subtrack"/>
```

Comment:
Codacy has a fix for the issue: [A space is required before closing bracket](https://app.codacy.com/gh/armors-dotnet/armors_control_panel/pullRequest?prid=8146859)
```suggestion
                                    <Image src={Subtrack} className="Subtrack" alt="Subtrack" />
```